pixiedust_node/node.py

```diff
@@ -13,11 +14,10 @@ class Node:
     # run a JavaScript script (path) with "node"
     def __init__(self, path):
         # get the home directory
-        node = 'node';
-        home = get_ipython().home_dir
+        home = Environment.pixiedustHome
```

Comment:
@glynnbird it would be cleaner if the pixiedust_node home was: os.path.join(Environment.pixiedustHome, "node").